dali/test/python/test_utils_tensorflow.py

```diff
@@ -36,15 +36,4 @@ def available_gpus():
     devices = []
     for device_id in range(num_available_gpus()):
         devices.append('/gpu:{0}'.format(device_id))
-    return devices
-
-
-def dataset_options():
-    options = tf.data.Options()
-    try:
-        options.experimental_optimization.apply_default_optimizations = False
-        options.experimental_optimization.autotune = False
-    except:
-        print('Could not set TF Dataset Options')
-
-    return options
\ No newline at end of file
+    return devices
```

Comment:
```suggestion
    return devices
    
```
Adding empty line at the end